src/string/grapheme.cr

```diff
@@ -39,6 +39,9 @@ class String
     end
   end
 
+  # Returns the number of Unicode extended graphemes clusters in this string.
+  #
+  # * `#each_grapheme` iterates the grapheme clusters.
```

Comment:
What's the purpose of this comment here?

Reply:
It's a reference to a related method.

I've used this in other places as well.